tests/end2end/cypress/integration/form_edition_value_relation_field-ghaction.js

```diff
@@ -0,0 +1,70 @@
+describe('Form edition all field type', function() {
+    before(function(){
+        // Runs before each tests in the block
+        cy.visit('/index.php/view/map/?repository=testsrepository&project=form_edition_value_relation_field')
+        // Start by launching feature form
+        cy.get('#button-edition').click()
+        cy.get('#edition-draw').click()
+    })
+
+    it('No expression menulist 4 values + 1 empty value', function () {
+        cy.get('#jforms_view_edition_code_without_exp option').should('have.length', 5)
+        cy.get('#jforms_view_edition_code_without_exp option').first().should('have.text', '')
+            .nextAll().each(($opt, index, $options) => {
+                cy.wrap($opt).should('not.have.text', '')
+            })
+    })
+
+    it('Simple expression menulist 2 values + 1 empty value', function () {
+        cy.get('#jforms_view_edition_code_with_simple_exp option').should('have.length', 3)
+        cy.get('#jforms_view_edition_code_with_simple_exp option').first().should('have.text', '')
+            .nextAll().each(($opt, index, $options) => {
+                cy.wrap($opt).should('not.have.text', '')
+            })
+    })
+
+    it('Parent field menulist 3 values + 1 empty value', function () {
+        cy.get('#jforms_view_edition_code_for_drill_down_exp option').should('have.length', 4)
+        cy.get('#jforms_view_edition_code_for_drill_down_exp option').first().should('have.text', '')
+            .nextAll().each(($opt, index, $options) => {
+                cy.wrap($opt).should('not.have.text', '')
+            })
+    })
+
+    it('Child field menulist 0 value + 1 empty value', function () {
+        cy.get('#jforms_view_edition_code_with_drill_down_exp option').should('have.length', 1)
+        cy.get('#jforms_view_edition_code_with_drill_down_exp option').first().should('have.text', '')
+    })
+
+    /*
+    it('Child field menulist after parent select', function () {
+        // Select A in parent field
+        cy.get('#jforms_view_edition_code_for_drill_down_exp').select('A').should('have.value', 'A')
+        cy.wait(100)
+        cy.get('#jforms_view_edition_code_with_drill_down_exp option').should('have.length', 3)
```

Comment:
I have no errors here.

Reply:
I have error in github actions.

Reply:
Link?

Reply:
It is failing again. Try `cy.wait(2000)` just to be sure delay is causing issue.

Reply:
I have not try 2000, only 800.